debugger/IController.hx

```diff
@@ -162,27 +162,24 @@ enum Command
     // ErrorEvaluatingExpression
 }
 
-
 /**
- * A list of strings
+ * An array of strings
  **/
-enum StringList
-{
-    Terminator;
-    Element(string : String, next : StringList);
-}
+typedef StringArray = Array<String>;
 
 
 /**
- * A list of class, possibly truncated so as not to be too large.
- * If truncated, a subsequent query for the remainder of the list can be done
- * using the continued expression.
+ * An array of classes. It's not being truncated. If this becomes an issue, it should
+ * be solved by adding a parameter that could be used to get the remainder of the array.
+ * It would have been much better if we had structs available in place of classes.
```

Comment:
Well reading a little more about haxe serialization, it says it supports classes, however I think that enums are better for serialization because there is less chance of holding state that is difficult for the serializer to recover.